### PR TITLE
ci: re-enable nix

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
Apparently it does still use Coq 8.17. What am I doing wrong? @CohenCyril

**Comment by CohenCyril** [comment]
you probably need to update the toolbox

**Comment by gares** [comment]
Thanks

**Comment by gares** [comment]
What is wrong with the mathcomp-classical job? I see no output :-/

File: .nix/config.nix [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
   format = "1.0.0";
   attribute = "coq-elpi";
-  default-bundle = "coq-8.17";
+  default-bundle = "coq-8.18";
   bundles = {
 
-    "coq-8.17".coqPackages = {
-      coq.override.version = "8.17";
+    "coq-8.18".coqPackages = {
       hierarchy-builder.override.version = "master";
```
Comment (gares) [inline .nix/config.nix:8]:
```suggestion
    "coq-8.18".coqPackages = {
      hierarchy-builder.override.version = "master";
      coq.override.version = "8.18";
```

**Pull request closed without merging**